test/system/030-run.bats

```diff
@@ -1233,4 +1233,18 @@ search               | $IMAGE           |
     done < <(parse_table "$tests")
 }
 
+@test "podman run umask" {
+    default_umask="0022"
+    umask="0017"
+    run_podman run -q $IMAGE grep Umask /proc/self/status
```

Comment:
you could exec `sh -c umask` which seems easier as we can then do an exact output match.

Reply:
umask does not exists in the ALPINE image.

Reply:
umask is a shell built-in, you need to use `sh -c umask` as mentioned:
```
$ podman run --rm quay.io/libpod/testimage:20221018 sh -c umask
0022
```